config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
Thanks @nenioscio, nice find!  I forgot you can specify bytes directly.  I like the idea of specifying the max bytes in one place and let the code figure out the details.

What do you think @bdunne?  It feels like there's other duplication above on the `127.0.0.1` although I don't think we've ever really configured those values before so refactoring this might be YAGNI.  I'd be ok with this proposed change with the intent to simplify this later on if people want to configure something else.  The original `memcache_server_opts` was added years ago for development so I could run with various verbosity levels.